components/contactus.php

```diff
@@ -1,32 +1,55 @@
-<link href="contactus.css" rel="stylesheet">
-<div class="container contact-form">
-    <form method="post">
-        <h3>Send us message </h3>
-       <div class="row">
-            <div class="col-md-6">
+<!DOCTYPE html>
+<html>
```

Comment:
Contact us is a component, so don't use a hole HTML template, just only use relevant elements.